### DISCUSSION TITLE
Adding carbon to the training setup

### DISCUSSION THREAD

**Discussion by photon-e** [codewars/codewars.com#2755] [Ideas]
Googles new language carbon is making waves now. i am sure it will eventually be added here as soon as posible.

**Comment by hobovsky** [comment]
Requests for new languages should be posted in Codewars Runner repository: https://github.com/codewars/runner

**Reply by photon-e** [reply]
ok, thank you.

**Comment by hobovsky** [comment]
Closing the discussion since the request has been posted in the runner repo here: https://github.com/codewars/runner/issues/198
